internal/plugin/server.go

```diff
@@ -505,13 +525,21 @@ func (plugin *NvidiaDevicePlugin) updateResponseForDeviceListEnvVar(response *pl
 // updateResponseForDeviceMounts sets the mounts required to request devices if volume mounts are used.
 func (plugin *NvidiaDevicePlugin) updateResponseForDeviceMounts(response *pluginapi.ContainerAllocateResponse, deviceIDs ...string) {
 	plugin.updateResponseForDeviceListEnvVar(response, deviceListAsVolumeMountsContainerPathRoot)
+
 	for _, id := range deviceIDs {
 		mount := &pluginapi.Mount{
 			HostPath:      deviceListAsVolumeMountsHostPath,
 			ContainerPath: filepath.Join(deviceListAsVolumeMountsContainerPathRoot, id),
 		}
 		response.Mounts = append(response.Mounts, mount)
 	}
+	for _, channel := range plugin.imexChannels {
+		mount := &pluginapi.Mount{
+			HostPath:      deviceListAsVolumeMountsHostPath,
+			ContainerPath: filepath.Join(deviceListAsVolumeMountsContainerPathRoot, "imex", channel.ID),
+		}
+		response.Mounts = append(response.Mounts, mount)
+	}
```

Comment:
This is contingent on adding support for this to the toolkit, but I guess there is vendoring needed here -- we just need to ensure we are running on a version of the toolkit that supports this.

What happens if someone is on a new plugin, but an old toolkit? I guess the IMEX channels just won't be injected? That seems fine., I think, so long as there is no error.

Reply:
OK. I think this may cause some churn, but I think its probably OK to say that if you want IMEX support with the plugin you need minimum version X of the toolkit.